File: wrapper/rules/fixtures/db/orm.js
module.exports = {
  up: (q) => q.createTable('widgets', { id: {} }),  // declaration + schema write
  down: (q) => q.dropTable('widgets'),              // schema write
};
const model = { modelName: 'Gadget', tableName: 'gadgets' };  // declaration

File: wrapper/rules/fixtures/db/orm.go
package widget

type TableName string

const (
	TbWidget TableName = "widgets"    // registry entry (literal)
	TbGadget TableName = "gadgets"    // registry entry (literal)
	TbAlias  TableName = OtherConst   // non-literal → unresolved go-const
)

func (w *Widget) TableName() string { return "widgets" }                   // literal declaration
func (g *Gadget) TableName() string { return constant.TbGadget.String() }  // resolves via registry
func (s *Sprocket) TableName() string { return dynamicName() }             // unresolved binding
